randovania/gui/lib/node_list_helper.py

```diff
@@ -80,8 +80,8 @@ def _on_check_world(c: QtWidgets.QCheckBox, _):
                 new_node_list = [
                     identifier
                     for area in areas if c.is_dark_world == area.in_dark_aether
-                    for node in nodes_by_area[area.name]
-                    if (identifier := NodeIdentifier(world_list.identifier_for_area(area), node)) in checks_for_node
+                    for node in nodes_by_area[self.game_description.world_list.identifier_for_area(area)]
```

Comment:
Just use the temp variable `world_list `:
`for node in nodes_by_area[world_list.identifier_for_area(area)]`